server.py

```diff
@@ -16,7 +16,7 @@
 def __is_session_valid(email, session_token):
```

Comment:
I also suggest that we introduce type hints for readability and understanding, e.g.:

```python
def __is_session_valid(email: str, session_token: str) -> bool:
```